sql/src/main/java/io/crate/protocols/postgres/BulkPortal.java

```diff
@@ -113,6 +114,14 @@ public Portal bind(String statementName,
                        @Nullable AnalyzedStatement analyzedStatement,
                        List<Object> params,
                        @Nullable FormatCodes.FormatCode[] resultFormatCodes) {
+        if (!this.query.equals(query)) {
```

Comment:
should this possibly be equals ignore case?

Reply:
@andreidan The ignore case could lead to problems if case sensitive identifiers within `"<ident>"` are used.

Reply:
@matriv yep figured as much (although the other unquoted case variations will be treated as different queries this way)